engine/src/CalciteExpressionParsing.cpp

```diff
@@ -258,42 +251,19 @@ cudf::type_id get_output_type_expression(const cudf::table_view & table, std::st
 			interops::operator_type operation = map_to_operator_type(token);
 			if(is_binary_operator(operation)) {
 				if(operands.size() < 2)
-					throw std::runtime_error(
-						"In function get_output_type_expression, the operator cannot be processed on less than one or "
-						"zero elements");
+					RAL_FAIL("In function get_output_type_expression, the operator cannot be processed on less than one or zero elements");
 
 				cudf::type_id left_operand = operands.top();
 				operands.pop();
 				cudf::type_id right_operand = operands.top();
 				operands.pop();
 
-				// TODO percy cudf0.12 was invalid here, should we consider empty?
-				if(left_operand == cudf::type_id::EMPTY) {
```

Comment:
porque se borro este codigo? Asumo que nunca deberia de suceder, pero si sucede por alguna razon no esperada, seria util obtener un error deterministico que nos ayude a saber rapidamente que paso

Reply:
that check now is done in `get_output_type`